src/main/java/io/sniffy/sql/StatementMetaData.java

```diff
@@ -60,4 +47,27 @@ private static Query guessQueryType(String sql) {
         return Query.OTHER;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method equals() has an NPath complexity of 240](https://www.codacy.com/app/sniffy/sniffy/file/2534793016/issues/source?bid=3005960&fileBranchId=3283367#l51)